Jenkinsfile_CNP

```diff
@@ -5,8 +5,5 @@
 def product = 'div'
 
 withInfraPipeline(product) {
-}
-
-onMaster {
-    enableSlackNotifications("#div-dev")
+    enableSlackNotifications(product)
```

Comment:
Sorry, it's wrong again.. 
we don't have a slack channel called 'div'

Reply:
😅